app/src/main/res/raw/init.js

```diff
@@ -40,6 +40,14 @@ window.AlphaWallet.init(rpcURL, {
     AlphaWallet.addCallback(id, cb)
     alpha.signTypedMessage(id, JSON.stringify(msgParams))
   },
+  ethCall: function (msgParams, cb) {
+    console.log("eth_call", msgParams)
+    const data = msgParams
```

Comment:
Looks like this line might not be needed

Reply:
Yes it's not needed, but is handy for the debug log and is consistent with the other functions.